oak_runtime/src/introspection_browser_client/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
Please look at https://google.github.io/styleguide/shellguide.html for the relevant style guidelines. In particular, the file should probably be renamed to not have the `.sh` extension.

Reply:
Good point! Will amend